frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_user_view.dart

```diff
@@ -389,7 +388,7 @@ class IconOption extends StatelessWidget {
       borderRadius: Corners.s6Border,
       hoverColor: Theme.of(context).colorScheme.tertiaryContainer,
       onTap: () => setIcon(iconUrl),
-      child: FlowySvg(emoji, size: _iconSize, overrideColor: false),
+      child: FlowySvg(emoji, size: _iconSize),
```

Comment:
Since there was no color provided, the color filter on the SVG widget was null anyway. We can just remove it here.